src/intel_sgx_ra/attest.py

```diff
@@ -50,14 +51,10 @@ def remote_attestation(quote: Union[Quote, bytes], base_url: str):
     if pck_platform_crl.get_revoked_certificate_by_serial_number(
         pck_platform_ca_cert.serial_number
     ):
-        logging.info(
-            "[ %4s ] Checking Intel PCK Platform certificate against CRL", "FAIL"
-        )
+        logging.info("[ %4s ] Check Intel PCK Platform certificate against CRL", "FAIL")
         raise CertificateRevokedError("Intel PCK Platform certificate revoked")
 
-    logging.info(
-        "[ %4s ] Checking certificates against Certificate Revocation List (CRL)", "OK"
-    )
+    logging.info("[ %4s ] Check Certificate Revocation List (CRL)", "OK")
```

Comment:
Put "OK" and "FAIL" in a global variable? Instead of copy/paste and risking typos as above.

Even `[ %4s ]`?